tools/src/main/java/org/apache/rocketmq/tools/command/namesrv/GetNamesrvConfigCommand.java

```diff
@@ -66,11 +69,11 @@ public void execute(final CommandLine commandLine, final Options options,
 
             Map<String, Properties> nameServerConfigs = defaultMQAdminExt.getNameServerConfig(serverList);
 
-            for (String server : nameServerConfigs.keySet()) {
+            for (Entry<String, Properties> serverEntry : nameServerConfigs.entrySet()) {
```

Comment:
serverEntry could be renamed to nameServerConfigEntry

Reply:
already edited